README.md

```diff
@@ -1 +1,20 @@
-# angular-using-behavioural-directives-lab
\ No newline at end of file
+# Using behavioural Directives
+
+## Objectives
+
+- Use both event and behavioural Directives
+- Create an input with ng-model
+- Display the input's value in the View
+- Render a list of DOM nodes using ng-repeat
+
+## Instructions
+
+Clone this repo and use the directory structure setup for you.
+
+In this lab, we've got a controller named `ContactController` with an array of contacts inside.
+
+We need these contacts to be repeated in our HTML with their name and phone number displayed.
+
+We should also have a button that calls our `addContact` method, to add a new contact to the list.
```

Comment:
Do we need inputs to enter name and phone number, or do you want the student to hard-code the new contact info for now?